frontend/src/components/SiteNav.js

```diff
@@ -56,6 +60,24 @@ const SiteNav = ({
     }
   }, [hasAlerts]);
 
+  // Determine Default Region.
+  const regions = allRegionsUserHasPermissionTo(user).join(', ');
+  const defaultRegion = user.homeRegionId || regions[0] || 0;
```

Comment:
`regions` ends up being a string here.

if the user has permissions to regions 11 and 12, `regions` will be the string `11, 12`. this means that if `user.homeRegionId` is falsy, then `defaultRegion` will end up being `1` (because `regions[0] accesses the character at position `0` in the string `11, 12`, which is `1`). what we probably want is for it to be 11.

maybe it makes sense to return the string to an array before accessing the first element, like:

```typescript
const defaultRegion = user.homeRegionId || regions.split(', ')[0] || 0;
```